doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -586,6 +586,12 @@ Other libraries
   * New library.
     This was previously a component of the :ref:`nrf53_audio_app` application, now moved to :file:`lib/pcm_mix`.
 
+* :ref:`app_event_manager`:
+
+  * Updated the way section names are created for event subscribers.
+    This allows you to use any event naming scheme.
+    For detailed information see the :ref:`NCSIDB-925 <ncsidb_925>` issue description on the :ref:`known_issues` page.
```

Comment:
```suggestion
    For more information, see the :ref:`NCSIDB-925 <ncsidb_925>` issue description on the :ref:`known_issues` page.
```

Reply:
@rakons double-checked this as discussed and the ref seems to be working just fine :)